open-craft/problem-builder
# [FAL-2198] Django 3.2 support

This updates the tests so that they run in an environment with Django 3.2.
It also fixes a few deprecation warnings.

Note: this also incorporates updates from https://github.com/open-craft/problem-builder/pull/332.

**Test instructions**:
1. Verify that the automated tests succeed on python3.8 / Django 3.2
2. Verify that the XBlock works correctly in xblock-sdk with Django 3.2
